docs/guide/piece-pages.md

```diff
@@ -1,257 +1,208 @@
+This feature set is powered by the `@apostrophecms/piece-page-type` module (because you're creating a *page type* that displays *pieces*, get it?).
 
-If the project has a piece module called `product`, then that module provides a way to create, edit, manage and query pieces. If the project has a piece page module called `product-page`, then that module provides a way to browse and view the pieces.
+Index pages support all features from [pages](/guide/pages.md), then add on some special features. In short, those are:
 
-In general piece modules are concerned with *editing and APIs*, while piece page modules are concerned with *browsing as part of a website*. For developers familiar with the "model / view / controller" pattern: **piece modules are the model layer for your content type, while piece page modules are the view layer.**
+- Two template files: one for index pages and one for show pages
+- Additional piece data available in templates
 
-Some projects just need a widget module for each piece, but most will want to let users view a page for each one, or at least browse and paginate through a complete list. That's where piece pages shine.
+## Creating a piece page type
 
-## Piece Pages by Example
+There are two critical steps to adding a new module for a piece page type:
 
-In A3, piece pages work much like 2.x, with a few important changes. Most of those changes are described by the the [new module format](module-format-example.md).
+1. Extend `@apostrophecms/piece-page-type`
+2. Specify what piece type should be shown on the page
 
-Just like in A2, piece page modules and piece modules come in pairs. And the name of the piece page module automatically determines which piece module it works together with. So we don't need very much code in our actual module in order to get started.
+Extending the right module is simple enough. These modules use the property:
 
-Here's an example of a piece page module that works with the product piece described in the [pieces section](pieces.md):
-
-```js
-// app.js
-require('apostrophe')({
-  modules: {
-    // ... other modules ...
-    // piece module
-    product: {},
-    // piece page module
-    'product-page': {}
-  }
-});
+```javascript
+`extend: '@apostrophecms/piece-page-type',`
 ```
 
-```js
-// modules/page/index.js
-// Add the new page type to the "Type" dropdown for new pages
+Identifying the piece type can be done two ways: **using a module naming convention** or **using the `pieceModuleName` setting**. We can look at both options using a blog as our example.
+
+### Matching a piece type using naming
+
+In this example, the piece type is `article`, since "articles" are the individual things that make up a blog. You can then name the piece page type as `article-page` and Apostrophe will automatically know that the two modules go together. (In case you missed the trick there, the piece page name is: piece type + `-page`.)
```

Comment:
```suggestion
In this example, the piece type is `article`, since "articles" are the individual entries that make up a blog. If you name the piece page type `article-page`, Apostrophe will automatically know that the two modules go together. (In case you missed the trick there, the piece page name is: piece type + `-page`.)
```